### DISCUSSION TITLE
Multiple Configs of Dyndns on Unifi

### DISCUSSION THREAD

**Discussion by UnInfamous** [willswire/unifi-ddns#64] [Q&A]
Hey all, this is excellent and I really appreciate the work done on this!

I'm attempting to add a second domain to Unifi for another domain, can't see there is any other configuration I need to do with Cloudflare. Because Unifi doesn't allow using the same ddns service twice (`dyndns`), and using `custom` as the second service doesn't work.

Anything you can think of to be able to get around this, whether it be the CLI or otherwise (without introducing another third-party service)?

**Comment by lobrau** [comment]
If you're trying to update multiple domains you can add the domains to update with a comma separated list.
ex: mydomain.com,*.mydomain.com

**Comment by jongroveher** [comment]
What if I want to edit different domains? Ie. 
`mydomain.com,*.mydomain.com,my2nddomain.com,*.my2nddomain.com`

What user(s) should I input?

**Comment by Toudahl** [comment]
I have the same problem. Multiple domains, with multiple subdomains.

Is it supported with this tool?

**Comment by willswire** [comment]
The latest release works with both the `custom` and `dyndns` providers for multiple domains.

**Reply by Toudahl** [reply]
Happy new year, William. I look forward to trying your update.
